string target_ip
uint16 port_index
---
uint8 grip_state
